lib/blockchain.py

```diff
@@ -420,6 +423,37 @@ def get_suitable_block_height(self, suitableheight, chunk=None):
 
         return blocks1['block_height']
 
+    _cached_asert_anchor: Optional[asert_daa.Anchor] = None  # cached Anchor, per-Blockchain instance
```

Comment:
Just so I understand right -- since this is per-Blockchain it means we never need to worry about the anchor changing (because blocks/headers are never rewinded) ? If so, awesome!

Reply:
Yes! That is correct! :D  Each instance of these is "per chain" -- so the problem we had in BCHN sources of invalidating the cached anchor for a different chain doesn't exist as a problem here (since each chain gets its own `Blockchain` instance).